imgix/urlbuilder.py

```diff
@@ -59,6 +62,11 @@ def __init__(
                           'Use `include_library_param` instead.',
                           DeprecationWarning, stacklevel=2)
 
+        if isinstance(domains, (list, tuple)) and (len(domains) > 1):
+            warnings.warn('Domain sharding has been deprecated and will ' +
+                          'be removed in the next major version. ',
+                          DeprecationWarning, stacklevel=2)
+
         if not isinstance(domains, (list, tuple)):
```

Comment:
As a follow up to my earlier comment, this line will eventually need to be removed or modified to convert from a list to a string. The consequence of doing it now would mean having to refactor how domain validation works. I can't decide if that's something that's better done now or at the point when domain sharding is removed.

Reply:
I think we should wait until the sharding is removed--this PR should *only* be concerned with adding a warning.